forc-test/src/ecal.rs

```diff
@@ -99,6 +104,10 @@ impl EcalHandler for EcalSyscallHandler {
                 let bytes = vm.memory().read(addr, count).unwrap().to_vec();
                 Syscall::Write { fd, bytes }
             }
+            FFLUSH_SYSCALL => {
```

Comment:
would be great to have a unit test for this

Reply:
Will be tested by the other `dbg` PR.

Reply:
Being tested by: https://github.com/FuelLabs/sway/pull/7119